reproman/resource/tests/test_session.py

```diff
@@ -394,19 +394,20 @@ def import_resource(mod, cls):
                    cls)
 
 
-@pytest.mark.parametrize(
-    "location",
-    [   # module, class
-        ("singularity", "SingularitySession"),
-        ("singularity", "PTYSingularitySession")
-    ],
-    ids=lambda x: x[1])
-def test_session_singularity(location, singularity_resource, check_methods):
-    """Test sessions that depend on `singularity_resource` fixture.
-    """
-    cls = import_resource(*location)
-    session = cls(singularity_resource.name)
-    check_methods(location[1], session)
+# TODO(asmacdo) we need to figure out how to work with apptainer (and singularity?)
```

Comment:
should they just be skipped if there is no singularity and otherwise run?

Reply:
May be, but I assume there is an "oldest singularity" we can support. Unfortunately when apptainer is installed, singularity is aliased to apptainer, but apptainer versions started over at 1.0.

Reply:
then skip if apptainer is installed, but generally should be ran if singularity (not apptainer'ed) is installed.

Reply:
skipped in latest commit